packages/graph/hash_graph/lib/graph/src/store/postgres/query/statement/select.rs

```diff
@@ -431,7 +440,7 @@ mod tests {
             &compiler,
             r#"
             WITH "entities" AS (SELECT *, MAX("entities"."version") OVER (PARTITION BY "entities"."entity_id") AS "latest_version" FROM "entities")
-            SELECT DISTINCT
+            SELECT
```

Comment:
(comment not for line), should we have a test for testing the new distinct compilation? Perhaps one for checking `DISTINCT ON(entity_id, version)` or something just to test with multiple columns?

Reply:
I added a new test in b929380c7e387faba6779ffe45e13c07aaa194d7